xbmc/utils/StreamDetails.h

```diff
@@ -86,6 +86,8 @@ class CStreamDetailSubtitle final : public CStreamDetail
   void Serialize(CVariant& value) const override;
   bool IsWorseThan(const CStreamDetail &that) const override;
 
+  std::string m_codec;
+  std::string m_type;
```

Comment:
Name: "type" is a generic word, maybe "location" is better?

type; non-localized strings "muxed" or "external" can work for storage but are not great for display to the user and I don't think translation should be up to the skin.

I don't know if there could ever be more values than internal/external > a bool could be enough. Or a number for an enum.

Reply:
Subtitle Type cannot be a boolean because this property will be used to know if there is or not a subtitle - and if you set it to false, what would that mean ? no subtitle or external one?. Codec cannot be used for this, because it is only retrieved if the subtitle is internal.

Reply:
The building has missed, any idea why ? I don't have any authorization to look at it.
This would be great if this PR goes ahead ;)

Reply:
Yeah I'll get back to this one once I free up my queue :')